backend/pycon/storage_backends.py

```diff
@@ -0,0 +1,12 @@
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+class StaticStorage(S3Boto3Storage):
+    location = "static"
```

Comment:
Hmm, do we want to use the same bucket for the static files? I would go for a different bucket and not a different path, bucket are free and I feel safer with media and static files separated.

What do you think @patrick91?